extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -578,15 +621,17 @@ export class SafeframeHostApi {
    * @param {number} height
    * @param {number} width
    * @param {string} messageType
-   * @param {boolean=} optIsCollapse
    */
-  resizeAmpAdAndSafeframe(height, width, messageType, optIsCollapse) {
+  resizeAmpAdAndSafeframe(height, width, messageType) {
+    const isCollapsed = messageType == SERVICE.COLLAPSE_RESPONSE;
+    const isReducing = messageType == SERVICE.COLLAPSE_RESPONSE ||
+          messageType == SERVICE.SHRINK_RESPONSE;
     // First, attempt to resize the Amp-Ad that is the parent of the
     // safeframe
     this.baseInstance_.attemptChangeSize(height, width).then(() => {
       // If this resize succeeded, we always resize the safeframe.
       // resizeSafeframe also sends the resize response.
-      this.resizeSafeframe(height, width, !!optIsCollapse, messageType);
+      this.resizeSafeframe(height, width, isCollapsed, messageType);
```

Comment:
Why do we need isCollapsed if we're passing messageType?

Reply:
removed.